rust-connectors/common/test-data/connectors/with_transform_many.yaml

```diff
@@ -0,0 +1,14 @@
+version: 0.1.0
```

Comment:
version bump?

Reply:
This is test data, so the actual version value is unimportant here.